AuxTel/Troubleshooting/General-Troubleshooting/AuxTel-Images-out-of-focus.rst

```diff
@@ -0,0 +1,117 @@
+	- If a section within the template is not needed, comment out the section title and label reference. Include a comment explaining why this is not required.
+    - If a file cannot include a title (surrounded by ampersands (#)), comment out the title from the template and include a comment explaining why this is implemented (in addition to applying the ``title`` directive).
+
+.. Include one Primary Author and list of Contributors (comma separated) between the asterisks (*):
+.. |author| replace:: *I. Sotuela*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *Carlos Morales, Karla Aubel, Erik Dennihy*
+
+.. This is the label that can be used as for cross referencing this procedure.
+.. Recommended format is "Directory Name"-"Title Name"  -- Spaces should be replaced by hyphens.
+.. _AuxTel-AuxTel-Troubleshooting-General-Troubleshooting-AuxTel-Image-out-of-focus-Procedure:
+.. Each section should includes a label for cross referencing to a given area.
+.. Recommended format for all labels is "Title Name"-"Section Name" -- Spaces should be replaced by hyphens.
+.. To reference a label that isn't associated with an reST object such as a title or figure, you must include the link an explicit title using the syntax :ref:`link text <label-name>`.
+.. An error will alert you of identical labels during the build process.
+
+#########################
+AuxTel Image out of focus
+#########################
+
+.. _Image-out-of-focus-Procedure-Overview:
+
+Overview
+========
+
+This procedure should be used when images appear out of focus due to a *WEP failure* or other related issues. Users will notice images ranging from slightly out of focus to large donut appearances when hexapod offsets are not correctly applied.
+
+.. _AuxTel-Image-out-of-focus-Procedure-Error-Diagnosis:
+
+Error diagnosis
+===============
+
+- The focus script :file:`latiss_wep_align` might fail, leaving the system out of focus.
+- These out of focus images might vary from slightly out of focus image when the WEP script took the system out of focus,
+  
+.. figure:: ./_static/min-focus.png
+    :name: in-focus
+    :width: 400
+
+    Slightly out-of-focus image
+
+to complete out-of-focus images that can appear as large donuts when hexapod offsets are left in intra or extra focus positions due to script failure.
+
+.. figure:: ./_static/out-focus.png
+    :name: out-focus
+    :width: 400
+
+    Wildly out-of-focus image
+
+.. _AuxTel-Image-out-of-focus-Procedure-Procedure-Steps:
+
+Procedure Steps
+===============
+
+.. _AuxTel-Image-out-of-focus-Procedure-Critical-Step-1:
+
+#. Pause ATScriptQueue.
+
+#. Clear all ATAOS offsets for the x, y, and z axes. Load and run the standard script :file:`auxtel/offset_ataos.py` **moving the script up in the queue** with the following configuration.
```

Comment:
:file:`auxtel/offset_ataos.py` inconsistent with previous usage. Maintain '.py' everywhere or remove it completely.

Reply:
Explain this using exact :guilabel:`` . 
e.g. In the script configuration window, toggle the the bar at the bottom of the window from :guilabel:`Bottom` to :guilabel:`Top`

Reply:
The details of the script queue will be described in a separate document that can definitely link here when available in our repo.